src/main/java/bio/terra/cloudfunctions/common/MediaTypeWrapper.java

```diff
@@ -0,0 +1,51 @@
+package bio.terra.cloudfunctions.common;
+
+import com.google.common.net.MediaType;
+
+public class MediaTypeWrapper {
+  // Workaround for Mime types not found in Guava MediaType
+  // See issue https://github.com/google/guava/issues/3946
+  public static final MediaType APPLICATION_GZIP = MediaType.create(MediaType.GZIP.type(), "gzip");
+  public static final MediaType APPLICATION_JSON =
+      MediaType.create(MediaType.JSON_UTF_8.type(), "json");
+
+  private MediaType mediaType;
+
+  /**
+   * MediaTypeWrapper constructor.
+   *
+   * @param input the Content-Type represented by the wrapper object, e.g. application/json
+   * @return MediaTypeWrapper object
+   */
+  public MediaTypeWrapper(String input) {
+    this.mediaType = MediaType.parse(input);
+  }
+
+  /**
+   * Check whether the MediaType represented by the wrapper object is equivalent to the provided
+   * parameter.
+   *
+   * @param mediaTypeRange Guava MediaType to compare to
+   * @return true/false
+   */
+  public boolean is(MediaType mediaTypeRange) {
+    assert (mediaTypeRange != null);
```

Comment:
Are we sure that assertions will be enabled in Google Cloud Functions?  I think it would be better to use something like `Preconditions.checkNotNull(mediaTypeRange)` that way we would not have to be sure the `-ea` flag is on for the GCF JVM invocation.

Reply:
Instead of `Preconditions.checkNotNull(mediaTypeRange)`, I use `Objects.requireNonNull(mediaTypeRange)` which will achieve the same effect.